components/ILIAS/UI/resources/js/Input/Field/file.js

```diff
@@ -337,9 +345,9 @@ il.UI.Input = il.UI.Input || {};
 			}
 
 			// abort if the given file is not an allowed file type.
-			if (dropzones[input_id].options.acceptedFiles &&
-				!Dropzone.isValidFile(file, dropzones[input_id].options.acceptedFiles)
-			) {
```

Comment:
I don't believe this change is necessary. Using `Dropzone.isValidFile()` here specifically addressed a mantis issue regarding mime-types.